Bottom: 18f2088923dcedae2201dbd7f20b86718f96d47d
Top:    546c6efb8cebb696270ff4f1d4df5f0af4e17d10
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-21 23:35:42 +0000

Refresh of polygon-h-reduce-circ_segs-to

---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
